guides/common/modules/con_limiting-resources.adoc

```diff
@@ -0,0 +1,18 @@
+[id="limiting-resources"]
+= Limiting resources
+
+You can use the Resource Quota plugin to limit users' access to resources.
```

Comment:
```suggestion
You can use the Resource Quota plugin to limit users' access to compute resources.
```
I'm missing the following information:
* What resources exactly? Is it limited to compute resources?
* In what use cases does the admin want to limit resource usage for users?
* How will it affect usage of compute profiles?

Reply:
"Resources" refers to the list of resources defined by the plugin: CPU, memory, and disk. It does not affect compute profiles which are merely a blueprint for the "size" of new hosts.

Reply:
A typical use case is to limit resource consumption for a user/a group of users, for example, when multiple departments rely on "one" infrastructure managed by one Foreman+Katello instance. ACK @bastian-src ?

Reply:
Exactly! Thanks @maximiliankolb and @Lennonka.
In short, the plugin enables administrators to limit the amount of resources users/usergroups can consume when provisioning hosts.

I see how the term is "resource" is a little difficult since its already used by compute resources. However, a resource quota relates to host resources as CPU cores, memory, and disk space - they are not directly related to a certain compute resource, but to users/usergroups and hosts.

Reply:
Yes, I have summarized it and rephrased it to "limiting host resources".